content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -152,7 +152,7 @@ fail validation.
 		<tr>
 			<td style="white-space: nowrap">Host Ports</td>
 			<td>
-				<p>HostPorts should be disallowed, or at minimum restricted to a known list.</p>
+				<p>HostPorts should be disallowed.</p>
```

Comment:
I think we need to keep a reference to the known list here as well if we're going to keep it in allowed values, but maybe make the statement more crisp?

```suggestion
				<p>HostPorts should be disallowed entirely or restricted to a known list.</p>
```

Reply:
We can be more opinionated and say "HostPorts should be disallowed entirely (recommended). At minimum, restrict HostPorts to a known list."

WDYT @liggitt?

Reply:
adding a `(recommended)` sounds good to me... updated my suggestion

Reply:
updated. PTAL